packages/shaper-utils/src/constants/Constant.ts

```diff
@@ -0,0 +1,3 @@
+export const Constant = {
+  NextJsConfig: 'next.config.js',
```

Comment:
1. `shaper-utils` cannot have any knowledge of plugins. Any plugin specific stuff should be sitting in the plugin.
2. Testing for plugin specific files is good, but when we do full snapshot testing of the generated directory, then checking of individual files should not be necessary. So let's shoot for the ultimate.

I will undo these changes and add additional concepts to make testing easier. Please hold off on making any changes to this branch.